src/components/AppFooter/AppFooter.tsx

```diff
@@ -16,8 +17,9 @@ import WhatsApp from './WhatsApp';
 import './AppFooter.scss';
 
 const AppFooter = () => {
-    const { currentLang = 'EN', localize, switchLanguage } = useTranslations();
+    const { localize, switchLanguage } = useTranslations();
     const { hideModal, isModalOpenFor, showModal } = useModalManager();
+    const currentLang = LocalStorageUtils.getValue<string>('i18n_language') || 'EN';
```

Comment:
instead of EN shouldn't it be currentLang from useTranslations?

Reply:
for the time being, im removing any instance for currentLang from useTranslations, the only reason why its default to EN for currentLang is because in the language provider, we pass the default language as EN. so for the time being, it is fine to just use 'EN' instead of currentLang from useTranslations